frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -204,6 +204,16 @@ class EditEmail extends Component {
     this.props.onChange({ ...this.state, reasonsForAction: newreasonsForAction });
   };
 
+  // triggers DOB tooltip button click in order to show/hide the tooltip window
+  triggerResponseTooltipClick = () => {
+    document.getElementById("your-response-tooltip-button").click();
+  };
+
+  // triggers DOB tooltip button click in order to show/hide the tooltip window
+  triggerReasonTooltipClick = () => {
+    document.getElementById("reasons-for-action-tooltip-button").click();
+  };
+
```

Comment:
Since the tooltips are now not opening on field select, these functions are not required anymore.
See next comment.

Reply:
removed in ea161a4